vmdb/app/models/ext_management_system.rb

```diff
@@ -153,7 +153,7 @@ def self.model_from_emstype(emstype)
 
   # UI methods for determining availability of fields
   def supports_port?
-    false
+    true
```

Comment:
This should not be done at the base level.  If you need port for kubernetes, then add an override method in EmsKubernetes.